tests/casp/stack_admin.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Start admin node (velum)
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use lockapi;
+
+sub run() {
+    # Wait until admin dashboard starts
+    my $timeout   = 240;
+    my $starttime = time;
```

Comment:
my $starttime = time;
my $timerun = time - $starttime;

so it will be time - time and what is time ?